model/ActivityMonitoringService.php

```diff
@@ -58,6 +85,13 @@ class ActivityMonitoringService extends ConfigurableService
         DeliveryExecution::STATE_FINISHIED,
     ];
 
+    protected $assessmentsDeliveryStatusesMaps = [
```

Comment:
I wonder if this would be better placed in `getRetiredDeliveries` function. This is opinion, so if you disagree that is acceptable. My argument would be that this is a one off property used in one place for one purpose.